word_guess.rb

```diff
@@ -0,0 +1,269 @@
+        @used_letters.push(@guess)
+    end
+
+    if @word.upcase.include?(@guess)
+      for index in 0..@word.length - 1
+        if @word[index].upcase == @guess
+          # update_word #took it out of method just to check variable
+          @word_so_far[index] = @guess
+        end
+      end
+      puts "What a great guess!"
+      puts "Good job!"
+      update_board
+    else
+      #guess is wrong
+      @wrong_count -= 1
+      puts "You have made a terrible guess!"
+      puts "How could you pick that letter?!"
+      update_board
+      puts "You have #{ @wrong_count } guesses left."
+    end
+  end
+
+  # this checks if you have more wrong guesses available
+  def more_tries?
+      @wrong_count >= 1
+  end
+
+  def word_done?
+    @word_so_far == @word.upcase
+  end
+
+  # this prints the winning game board
+  def win_game
+    #call board drawing
+    @wrong_count = 6
+    update_board
+    message = "YAY! YOU WIN! Your prize is an "
+    message2 = "#{@good_prize}"
+    puts message.colorize(:yellow) + message2.colorize(:magenta)
+    system "say", message
+    system "say", message2
+    exit
+  end
+
+  # this prints the losing game board
+  def lose_game
+    #call board drawing
+    wrong_count = 0
+    # update_board
+    message = "OH, NO! YOU LOSE!! Your prize is #{@bad_prize}..."
+    puts message.colorize(:red)
+    puts "The correct word was: #{@word.colorize(:yellow)}"
+    system "say", message
+    exit
+  end
+
+  # this prints the game boards and updates them.
+  def update_board
+    ghost_one = "=======".colorize(:cyan)
+    ghost_two = "   .-. "
+    ghost_thr = "  |OO |"
+    ghost_fou = "  |   |"
+    ghost_fiv = "  '^^^'"
+    ghost_six = "=======".colorize(:cyan)
+
+    pac_man_one = "=========".colorize(:cyan)
+    pac_man_two = "    .--. ".colorize(:yellow)
+    pac_man_thr = "   / _.-'".colorize(:yellow)
+    pac_man_fou = "   |  '-.".colorize(:yellow)
+    pac_man_fiv = "    '--' ".colorize(:yellow)
+    pac_man_six = "=========".colorize(:cyan)
+
+    dot_one = "=========".colorize(:cyan)
+    dot_two = "         "
+    dot_thr = "   .''.  "
+    dot_fou = "   '..'  "
+    dot_fiv = "         "
+    dot_six = "=========".colorize(:cyan)
+
+    case @wrong_count
+    when 6
+      puts pac_man_one + pac_man_one + pac_man_one + pac_man_one + pac_man_one
+      puts pac_man_two.blink + pac_man_two.blink + pac_man_two.blink + pac_man_two.blink + pac_man_two.blink
+      puts pac_man_thr.blink + pac_man_thr.blink + pac_man_thr.blink + pac_man_thr.blink + pac_man_thr.blink
+      puts pac_man_fou.blink + pac_man_fou.blink + pac_man_fou.blink + pac_man_fou.blink + pac_man_fou.blink
+      puts pac_man_fiv.blink + pac_man_fiv.blink + pac_man_fiv.blink + pac_man_fiv.blink + pac_man_fiv.blink
+      puts pac_man_six + pac_man_six + pac_man_six + pac_man_six + pac_man_six
+      puts @word_so_far
+
+    when 5
+      puts pac_man_one + dot_one + dot_one + dot_one + dot_one
```

Comment:
Consider using the `*` operator on these strings. This could simplify these long string concatenations.
This line here could be:
`puts pac_man_one + (dot_one * 4)`